ruma-events/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Breaking changes:
 * Use `ruma_identifiers::{ServerName, ServerKeyId}` in `signatures` fields of
   `pdu::RoomV1Pdu, RoomV1PduStub, RoomV3Pdu, RoomV3PduStub}` and
   `room::member::SignedContent`.
+* Remove origin field from PDUs (see https://github.com/matrix-org/synapse/issues/3816)
```

Comment:
This changelog entry should be in the first commit, and mention the feature gate (or be removed completely).